src/bower/PackageUtils.js

```diff
@@ -48,7 +48,8 @@ define(function (require, exports, module) {
      * @return {boolean}
      */
     function isValidDependencyType(type) {
-        return (type === DependencyType.DEVELOPMENT || type === DependencyType.PRODUCTION);
+        return (type === DependencyType.DEVELOPMENT || type === DependencyType.PRODUCTION ||
+                type === DependencyType.UNKNOWN);
```

Comment:
Perhaps it would look better (more readable and nicer) as
```javascript
switch (type) {
case DependencyType.DEVELOPMENT:
case DependencyType.PRODUCTION:
case DependencyType.UNKNOWN:
    return true;
default:
    return false;
}
```